scripts/cleanup_old_vms.py

```diff
@@ -71,19 +79,48 @@ def get_vm_config_modified_time(name, vm_name, datastore_url, provider_key):
         return False
 
 
-def process_provider_vms(provider_key, provider_type, matchers, delta, vms_to_delete):
+def list_provider_vms(provider_key):
+    try:
+        provider = get_mgmt(provider_key)
+        vm_list = provider.list_vm()
+        provider_type = cfme_data.get("management_systems", {})[provider_key].get('type', None)
+        now = datetime.datetime.now()
+        for vm_name in vm_list:
+            with open(args.outfile, 'a+') as report:
+                creation = provider.vm_creation_time(vm_name)
+                message = "{provider_type}:{provider}  {instance} " \
+                          "\t {time} \t {instance_type} \t {instance_status}\n\n" \
```

Comment:
any intent for humans reading this output, or is it being structured to be machine parsed? If human readable, add some labels please.